src/routes/index.jsx

```diff
@@ -131,6 +132,7 @@ const CustomRoutes = () => {
         <Route path="register" element={<Register />} />
       </Route>
       <Route path="*" element={<Navigate to="/app" />} />
+      <Route path="teacher" element={<Teacher />} />
```

Comment:
Put it this way instead
`
      <Route path="teacher" element={<Teacher />} />
      <Route path="*" element={<Navigate to="/app" />} />
`